### PR TITLE
Fix errors in Windows documentations

### PR DESCRIPTION
**Pull request by jxyang**
This PR adds back Windows 10 as a supported platform. It also fixes a few statements in windows get started documents that are no longer true after PR #2250.

### CODE REVIEW

File: docs/GettingStartedDocs/Contributors/WindowsInstallInfo.md [hunk 1]
```diff
@@ -5,8 +5,17 @@ You can locally install the SDK from the compiled Open Enclave tree by specifyin
...
+```cmd
+cmake .. -G  Ninja -DNUGET_PACKAGE_PATH=c:\your\path\to\intel_and_dcap_nuget_packages -DCMAKE_INSTALL_PREFIX:PATH=C:\openenclave
+ninja install
+```
+
+For SGX1 targets:
+
 ```cmd
-cmake .. -G  Ninja -DNUGET_PACKAGE_PATH=c:\your\path\to\intel_and_dcap_nuget_packages -DCMAKE_INSTALL_PREFIX:PATH=C:\openenclave -DHAS_QUOTE_PROVIDER=ON
+cmake .. -G  Ninja -DNUGET_PACKAGE_PATH=c:\your\path\to\intel_and_dcap_nuget_packages -DCMAKE_INSTALL_PREFIX:PATH=C:\openenclave -DHAS_QUOTE_PROVIDER=OFF
```
Comment (johnkord) [inline docs/GettingStartedDocs/Contributors/WindowsInstallInfo.md]: I am so happy the default configuration is ON for both platforms now!

**Comment by jxyang** [comment]
bors try

**Comment by radhikaj** [comment]
thanks for making the default on.

**Pull request merged**